submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/web/handlers/SubmissionWebHandler.kt

```diff
@@ -54,9 +50,22 @@ class SubmissionWebHandler(
 
     fun deleteSubmission(accNo: String, user: SecurityUser): Unit = submissionService.deleteSubmission(accNo, user)
 
-    private fun rootPath(submission: String, format: SubFormat) =
+    private fun getRootPath(submission: String, format: SubFormat) =
         serializationService.deserializeSubmission(submission, format).rootPath.orEmpty()
 
+    internal fun getComposedFilesSource(
```

Comment:
Why internal and not private?

Reply:
To be able to stub it.

Reply:
mmm it looks for me that we need to introduce then a source helper or something (out of the scope of this ticket) but break encapsulation for testing is the last resource IMO and 99.99% smell of a problem in design.